src/ansys/fluent/core/codegen/datamodelgen.py

```diff
@@ -66,6 +66,34 @@
 )
 
 
+def _write_meshing_utilities_stub(file_path):
+    if file_path.exists():
+        file_path.unlink()
+    file = open(file_path, "w", encoding="utf8")
```

Comment:
Done. Thank you.

Reply:
@hpohekar Please investigate to understand why the above error is coming. Then you should be able to make any corrections so that you can follow the advice given by @mkundu1.

Reply:
@seanpearsonuk AFAICS the file goes out of scope once we write initial few lines using `with` syntax, as we are returning file on last line which is outside of `with` block.

Reply:
@seanpearsonuk We can see that in `write_static_info` we are passing file opened using `with` syntax within that `with` block itself (line 426) which is not the case for `_write_meshing_utilities_stub`.

Reply:
@seanpearsonuk @mkundu1 We are using `with` syntax now. Thank you.